src/main/java/io/github/opencubicchunks/cubicchunks/mixin/core/common/progress/MixinLoggingChunkStatusListener.java

```diff
@@ -1,21 +1,34 @@
 package io.github.opencubicchunks.cubicchunks.mixin.core.common.progress;
 
+import io.github.opencubicchunks.cubicchunks.chunk.IBigCube;
 import io.github.opencubicchunks.cubicchunks.chunk.ICubeStatusListener;
 import io.github.opencubicchunks.cubicchunks.chunk.util.CubePos;
-import net.minecraft.util.math.ChunkPos;
+import net.minecraft.util.math.MathHelper;
 import net.minecraft.world.chunk.ChunkStatus;
 import net.minecraft.world.chunk.listener.LoggingChunkStatusListener;
 import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
 
 import javax.annotation.Nullable;
 
 @Mixin(LoggingChunkStatusListener.class)
 public abstract class MixinLoggingChunkStatusListener implements ICubeStatusListener {
+    private static final int CUBES_EDGE_LENGTH = ((((int) Math.ceil(10 * (16 / (float) IBigCube.BLOCK_SIZE)))*2)+3);
```

Comment:
Can you maybe elaborate on where these magic numbers come from? 10, 16, 2 and 3? If possible use constants from elsewhere in the code or reference the server view distance if possible. Otherwise adding a comment might clear things up.

Reply:
10 is the server view distance
16 is vanilla chunk size
I'll add a comment for the *2+3, but what constants are there for those two values?

Reply:
Actually wait, the 16 isn't necessary, I'll just use `CUBE_DIAMETER` instead of `BLOCK_SIZE`

Reply:
*2 is giving diameter instead of radius, and the +3 is including the centre cube as well as +1 on either side

Reply:
Isn't the 10 the parameter given to the constructor in `LoggingChunkStatusListener`? It takes the radius as its argument. This would require turning the constant into a normal field, mixin into the constructor and assigning it from there. But as the code is right now, it might not work properly with non-default server view distances.
Also, this might need adjusting once #538 is merged.